changes/ticket33072

```diff
@@ -0,0 +1,3 @@
+  o Minor bugfix (directory authority):
+    - Always allow compressed directory requests. Fixes bug 33072; bugfix on
```

Comment:
Should the changes file mention the new AuthDirRejectUncompressedRequests option?